TRANSCRIPTION.md

```diff
@@ -5,122 +5,88 @@
 
 | Segment | Occurrence | BIPA | CLTS SoundClass |
 |:----------|-------------:|:-------|:------------------|
-| a | 585 | ✓ | ✓ |
-| i | 470 | ✓ | ✓ |
+| a | 593 | ✓ | ✓ |
+| i | 473 | ✓ | ✓ |
 | k | 207 | ✓ | ✓ |
 | n | 202 | ✓ | ✓ |
-| e | 178 | ✓ | ✓ |
-| á | 166 | ✓ | ✓ |
-| í | 151 | ✓ | ✓ |
-| u | 144 | ✓ | ✓ |
-| h | 138 | ✓ | ✓ |
+| e | 179 | ✓ | ✓ |
+| á/a | 166 | ✓ | ✓ |
+| h | 153 | ✓ | ✓ |
+| í/i | 153 | ✓ | ✓ |
+| u | 147 | ✓ | ✓ |
 | m | 129 | ✓ | ✓ |
 | p | 119 | ✓ | ✓ |
+| ʔ | 119 | ✓ | ✓ |
+| w | 104 | ✓ | ✓ |
 | r | 97 | ✓ | ✓ |
-| w | 96 | ✓ | ✓ |
-| ʔ | 81 | ✓ | ✓ |
 | l | 74 | ✓ | ✓ |
-| ú | 68 | ✓ | ✓ |
+| ú/u | 68 | ✓ | ✓ |
 | t | 66 | ✓ | ✓ |
-| j | 59 | ✓ | ✓ |
-| o | 57 | ✓ | ✓ |
-| é | 53 | ✓ | ✓ |
-| ó | 46 | ✓ | ✓ |
+| j | 60 | ✓ | ✓ |
+| o | 59 | ✓ | ✓ |
+| é/e | 54 | ✓ | ✓ |
+| ó/o | 46 | ✓ | ✓ |
 | d | 45 | ✓ | ✓ |
+| ʃ | 43 | ✓ | ✓ |
 | b | 39 | ✓ | ✓ |
-| ʃ | 36 | ✓ | ✓ |
-| ɾ | 32 | ✓ | ✓ |
+| ɾ | 33 | ✓ | ✓ |
 | aa | 29 | ✓ | ✓ |
 | tʃ | 27 | ✓ | ✓ |
 | ii | 20 | ✓ | ✓ |
+| pʰ | 20 | ✓ | ✓ |
 | s | 20 | ✓ | ✓ |
-| à | 18 | ✓ | ✓ |
+| à/a | 18 | ✓ | ✓ |
 | g | 17 | ✓ | ✓ |
 | v | 17 | ✓ | ✓ |
-| ? | 16 | ? | ? |
 | ee | 16 | ✓ | ✓ |
-| ( | 13 | ? | ? |
 | ts | 13 | ✓ | ✓ |
-| ) | 12 | ? | ? |
+| ɲ | 13 | ✓ | ✓ |
 | a: | 12 | ✓ | ✓ |
 | i: | 12 | ✓ | ✓ |
-| ph | 12 | ✓ | ✓ |
-| nh | 10 | ✓ | ✓ |
-| ì | 10 | ✓ | ✓ |
-| ù | 9 | ✓ | ✓ |
-| 'a | 8 | ✓ | ✓ |
+| tsʰ | 10 | ✓ | ✓ |
+| ì/i | 10 | ✓ | ✓ |
+| kʰ | 9 | ✓ | ✓ |
+| ù/u | 9 | ✓ | ✓ |
 | dz | 8 | ✓ | ✓ |
 | e: | 8 | ✓ | ✓ |
-| pʰ | 8 | ✓ | ✓ |
 | y | 8 | ✓ | ✓ |
-| è | 8 | ✓ | ✓ |
-| é: | 8 | ✓ | ✓ |
-| kh | 7 | ✓ | ✓ |
-| tsʰ | 7 | ✓ | ✓ |
+| è/e | 8 | ✓ | ✓ |
+| é:/eː | 8 | ✓ | ✓ |
 | uu | 7 | ✓ | ✓ |
 | oo | 6 | ✓ | ✓ |
 | tt | 6 | ✓ | ✓ |
 | _ | 5 | ✓ | ✓ |
```

Comment:
please replace underscore by a `+`. We do no longer use `_` as a segmentation marker.